### ISSUE TITLE
Overuse of <p class="discreet"> 

### ISSUE THREAD

**Issue by 3dogMcNeill** [plone/Products.CMFPlone#792]
At least half of the text in a new instance is discreet (Welcome page, site setup pages), which should be used sparingly. Despite passing for WCAG AA at normal size with weight 300 and color #696969, it's difficult to see. More contrast would be better. Not using the discreet class would be even better.

**Comment by agitator** [comment]
+1 on more contrast

**Comment by davilima6** [comment]
Since [Plone 4.4 takes presentation mode](https://dev.plone.org/ticket/13270) [out of core](https://pypi.python.org/pypi/plone.app.s5slideshow), +1 on removing all but 1st discreet classes in Welcome page.

However in Site setup I counted only 2 occurrences of discreet and IMO both valid because they act similarly to formHelp class, ie: providing contextual help messages.

**Comment by 3dogMcNeill** [comment]
@davilima6 I meant on the pages within site setup, but I should have been more clear and requested increased contrast for .formhelp, which has even less contrast at #666666 and is used throughout the site setup pages (much more than discreet). Added an image to better show what I mean. 

![example](https://cloud.githubusercontent.com/assets/7101218/9097616/919d7290-3b92-11e5-8e49-a6fe792a464d.png)

**Comment by davilima6** [comment]
Both .formHelp and select box have more contrast than .discreet, they're actually brighter: 47%, 41% and 40%, respectively.

However since body background is at 98% and [W3C recommends](http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) 4.5:1 minimum contrast for font-size < 18px, our limit at 14px is 45% of luminosity (#737373) so .discreet does fail AA check: http://webaim.org/resources/contrastchecker/

For greater safety I'm +1 on limiting all text at least on #666 (no higher numbers) everywhere background is #fafafa. Also +1 on normalizing all too similar color choices because poor contrast makes for greater cognitive effort. It's like having font-size: 12px and 13px in the same page. Better go for 14px at least.

**Comment by jensens** [comment]
I think this was solved with barceloneta upgrades in Plone 6. Please reopen if I am wrong.
